server/adaptors/integrations/__data__/repository/apache/schemas/communication-1.0.0.mapping.json

```diff
@@ -5,7 +5,8 @@
                 "version": "1.0.0",
                 "catalog": "observability",
                 "type": "logs",
-                "component": "communication"
```

Comment:
Why is there a different number of files changes compared to the 2.x/2.9 PR? Can we double check that?

Reply:
this is due to deleting redundant file in 2.9 that were accidentally auto - merged in
this didnt happened in main branch